src/shaders/_prelude_fog.vertex.glsl

```diff
@@ -14,6 +15,12 @@ float fog_opacity(float t) {
     return u_fog_opacity * min(1.0, 1.00747 * falloff);
 }
 
+// This function much match fog_sky_blending defined in _prelude_fog.fragment.glsl
+float fog_sky_blending(vec3 camera_dir) {
```

Comment:
Need account for this in CPU side code as well.

Reply:
I think either direction is defensible, but yeah, I agree, @arindam1993. I've updated the CPU side code to include this.